__mocks__/react-native-key-command.ts

```diff
@@ -0,0 +1,9 @@
+import type {addListener as _addListener, constants as _constants} from 'react-native-key-command';
+
+const registerKeyCommands = () => {};
+const unregisterKeyCommands = () => {};
```

Comment:
What about these functions? Should we add them to `react-native-key-command.d.ts` and add typing in this file?

Reply:
Yes, let's add them to `react-native-key-command.d.ts` @kubabutkiewicz